charts/gsp-cluster/requirements.lock

```diff
@@ -10,7 +10,7 @@ dependencies:
   version: 6.4.1
 - name: kiam
   repository: https://kubernetes-charts.storage.googleapis.com/
-  version: 2.2.0
+  version: 2.2.5
```

Comment:
why didn't the digest in this file change? 😕

Reply:
It did this time...
🤷‍♀